gateway/engine/core/src/main/java/io/apiman/gateway/engine/impl/DefaultLdapSearchImpl.java

```diff
@@ -58,6 +56,10 @@ private void getResults(String searchDn, String filter, LdapSearchScope scope,
             List<SearchResultEntry> searchResults = connection.search(searchDn, searchScope, filter).getSearchEntries();
             result.handle(AsyncResultImpl.create(searchResults));
         } catch (LDAPException e) {
+            if (ldapErrorHandler == null) {
+                // TODO wire in logger
+                System.err.println("LDAP Error Handler not set. Error may be swallowed; this is probably not what you intended.");
+            }
```

Comment:
Apparently if you click resolve with a comment, the comment is not saved. 

Anyway, second try...

Yes, logging is tricky because we try to work on a variety of platforms without interfering with the underlying logging config. We need to try to crack it. I left the TODO so we can sweep it up later.